website/source/docs/configuration/ui/index.html.md

```diff
@@ -62,6 +62,21 @@ the Consul service address (if using Consul):
 https://vault.service.consul:8200/ui
 ```
 
+## Policy to browse secrets
+For a user to browse secrets from the k/v backend using the UI, they need to have read and list access to the secret mount, and also have read and list access to sys/mounts.
+
+Example:
+
+```hcl
+path "secret/*" {
+    capabilities = ["read","list"]
+}
+
+path "sys/mounts" {
```

Comment:
Access to `sys/mounts` hasn't been necessary for the UI since 0.10.1 - are you still seeing this on current versions of Vault?

Reply:
Yup, I was using 0.10.0. Newer versions don't need it. Thanks!